core/dronecore.cpp

```diff
@@ -38,6 +39,26 @@ DroneCore::ConnectionResult DroneCore::add_udp_connection(int local_port_number)
     return DroneCore::ConnectionResult::SUCCESS;
 }
 
+DroneCore::ConnectionResult DroneCore::add_tcp_connection()
+{
+    return add_tcp_connection("127.0.0.1", 0);
+}
+
+DroneCore::ConnectionResult DroneCore::add_tcp_connection(std::string ipAddress,
```

Comment:
I vote to keep variable naming consistent with underscores, so I suggest `ip_address` instead of `ipAddress`.